### PR TITLE
[WW-5371] Documents new Action File Upload Interceptor

### PR DESCRIPTION
**Pull request by lukaszlenart**
Refs [WW-5371](https://issues.apache.org/jira/browse/WW-5371)

### CODE REVIEW

File: source/core-developers/action-file-upload-interceptor.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+```java
+public class UploadAction extends ActionSupport implements UploadedFilesAware {
+   private UploadedFile uploadedFile;
+   private String contentType;
+   private String fileName;
+   private String originalName;
+
+   @Override
+   public void withUploadedFiles(List<UploadedFile> uploadedFiles) {
+      if (!uploadedFiles.isEmpty() > 0) {
```
Comment (kusalk) [inline source/core-developers/action-file-upload-interceptor.md:85]: Think you can drop the `> 0`

File: source/core-developers/file-upload.md [hunk 2]
```diff
@@ -51,7 +55,35 @@ example:
...
+public class UploadAction extends ActionSupport implements UploadedFilesAware {
+  
+    private UploadedFile uploadedFile;
+    private String contentType;
+    private String fileName;
+    private String originalName;
+
+    @Override
+    public void withUploadedFiles(List<UploadedFile> uploadedFiles) {
+        if (!uploadedFiles.isEmpty() > 0) {
```
Comment (kusalk) [inline source/core-developers/file-upload.md:71]: Same here

File: source/core-developers/file-upload.md [hunk 3]
```diff
@@ -21,11 +24,13 @@ than the temporary directory and the directories that belong to your web applica
...
 
+> NOTE: Since Struts 6.4.0 the `FileUploadInterceptor` is deprecated and you should use `ActionFileUploadInterceptor` instead!
+
 ## Basic Usage
 
-The `org.apache.struts2.interceptor.FileUploadInterceptor` class is included as part of the `defaultStack`. As long as
-the required libraries are added to your project you will be able to take advantage of the Struts 2 file upload
-capability. Configure an Action mapping for your Action class as you typically would.
+The `org.apache.struts2.interceptor.FileUploadInterceptor` and `org.apache.struts2.interceptor.ActionFileUploadInterceptor`
+classes is included as part of the `defaultStack`. As long as  the required libraries are added to your project you will be able
```
Comment (kusalk) [inline source/core-developers/file-upload.md:32]: `classes **are** included`

Comment (lukaszlenart) [reply source/core-developers/file-upload.md]: Rephrased the sentence  a bit

**Review by kusalk — APPROVED** [review]
Nice

**Pull request merged**